hw1/01-selectors.md

```diff
@@ -2,6 +2,34 @@
 
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple
+5. #fancy pickel
+6. .small
+7. orange.small
+8. bento orange.small
+9. plate, bento
+10. *
+11. plate *
+12. plate + apple
+13. bento ~ pickle
+14. plate > apple
+15. plate > orange:first-child
+16. plate apple:only-child, plate pickle:only-child
```

Comment:
Hi Stuart,
For level 16, you nailed it but I think we can improve it by just using "plate *:only-child"

Reply:
Ah nice, good catch!

Reply:
Updated the PR